### PR TITLE
feat: add SEO tags

### PR DESCRIPTION
**Pull request by orlando**
**What:** Adding social SEO tags.

**Why:** To render properly when sharing the link in social networks 

**How:** Adding variables to gatsby configuration file and using it with React Helmet

### Media
_note: the URL will show correctly when deploying to the actual domain_

image

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -29,10 +33,8 @@ const config = {
     {
       resolve: `gatsby-plugin-google-analytics`,
       options: {
-        trackingId: process.env.GOOGLE_ANALYTICS_TRACKING_ID,
         head: true,
-        sampleRate: 5,
```
Comment (duranmla) [inline gatsby-config.js]: Why do we have it and why remove it?

Comment (orlando) [reply gatsby-config.js]: This is optional and seems to be causing issues with the tracking. I'll move that to another commit.

**Comment by orlando** [comment]
It’s Gatsby, server side rendered react

**Comment by orlando** [comment]
Closed https://github.com/debtcollective/to-do/issues/6

**Pull request merged**